.github/workflows/build.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Setup .NET
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: '3.1.x'
+          dotnet-version: '6.0.x'
```

Comment:
Can you apply the same change to the workflows `deploy-website.yml` and `test-deploy-website.yml`?